pnpm-lock.yaml

```diff
@@ -1134,6 +1134,10 @@ packages:
     resolution: {integrity: sha512-2lpvuQn/qymQPfwR2SxLyRy/Wi/RrEYpbQyoc9SYfhartw9TBY8c34yZkd8zNU7Y/KG3h+PLrCmNpncocuB3YA==}
     engines: {node: '>=16.0.0'}
 
+  '@aws-sdk/client-s3@3.735.0':
```

Comment:
does this pnpm-lock.yaml makes sense @IrakliJani  ?

Reply:
suspiciously big diff for a package upgrade. I'll test it locally